Cargo.toml

```diff
@@ -5,6 +5,7 @@ members = [
     "packages/real/crates/real-base",
     "packages/hoprd/crates/hoprd-misc",
     "packages/utils/crates/utils-misc",
+    "packages/utils/crates/utils-proc-macros",
```

Comment:
Would it be possible to also include ordinary declarative macros in this crate?
Then we could call it just `utils-macros` and move there also stuff like `ok_or_str!` ...etc.

Reply:
Unfortunately not supported by the compiler